tms_objects2.erb

```diff
@@ -521,6 +521,17 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
         <% end %>   <%# BIBcitations |bc| %>
           </lido:event>
         </lido:eventSet>
+        <% unless DisplayName.nil? %>
```

Comment:
use Owners[0]["DisplayName"] if it works, rather than definition a specific variable